pkg/acsengine/defaults.go

```diff
@@ -238,3 +247,27 @@ func certGenerationRequired(a *api.Properties) bool {
 		return false
 	}
 }
+
+// getFirstConsecutiveStaticIPAddress returns the first static IP address of the given subnet.
+func getFirstConsecutiveStaticIPAddress(subnetStr string) string {
+	_, subnet, err := net.ParseCIDR(subnetStr)
+	if err != nil {
+		return DefaultFirstConsecutiveKubernetesStaticIP
+	}
```

Comment:
I updated the commit to do the subnet checks during validation phase, in validate.go. I also added corresponding test cases in validate_test.go.

I'm still returning the default here, just to cover the case when the hardcoded default in const.go is somehow not a valid subnet. I did not want to pollute the defaults logic here with validation code which also requires updating these functions to return error codes all the way up to main().